### PR TITLE
Dance Page -  videos section

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is to display the creativity video, playlist and inspirational videos.

Implementation:
- Added a creativity_video component to display creativity video
- Added  inspirational_video component to display inspirational videos
- Added  image thumbnail that links to the creativity playlist series.

Future work post hoc
- [ ] Refactor inspirational video component and display video thumbnail to display a playlist

### Screen shot - First row is the creativity video and playlist thumbnail, all other subsequent rows are inspirational videos.
screen shot 2018-11-06 at 8 59 20 pm

### CODE REVIEW

**Comment by maddiedierker** [comment]
@nkiruka just to confirm, the code we're reviewing here pairs with solution 1, right?

**Comment by nkiruka** [comment]
Yes

File: pegasus/sites.v3/code.org/views/dance_videos.haml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- videos.each do |video|
+  .col-50{style: "float:left; padding:10px"}
+    - url = "https://www.youtube.com/watch?v=#{video[:code]}"
+    - facebook = {u: url}
+    - twitter = {url: url, related: 'codeorg', text: "#{video[:text]} (Thanks #{get_random_donor_twitter} for supporting @codeorg)"}
+    =view :display_video_thumbnail, id: video[:id], video_code: video[:code], caption: "#{video[:text]} (#{video[:duration]})", play_button: 'center', facebook: facebook, twitter: twitter, download_path: video[:download]
+
+%div{style: 'clear: both'}
+
+.video-smalltext.hide
```
Comment (maddiedierker) [inline pegasus/sites.v3/code.org/views/dance_videos.haml]: note from in-person review: delete the footer text

File: pegasus/sites.v3/code.org/public/dance.haml [hunk 2]
```diff
@@ -17,6 +17,14 @@
 %h2= I18n.t(:hoc2018_dance_featured_student_creations_title)
 =view :student_dance_projects
 
-%h1= I18n.t(:hoc2018_dance_videos_section_heading)
+%h2= I18n.t(:hoc2018_dance_videos_section_heading)
+.starwars-videos{style: "clear: both;"}
```
Comment (maddiedierker) [inline pegasus/sites.v3/code.org/public/dance.haml]: note: change from starwars to dance

File: pegasus/sites.v3/code.org/public/dance.haml [hunk 3]
```diff
@@ -17,6 +17,14 @@
...
+  =view :dance_creativity_video
+  %a{href: "https://www.youtube.com/playlist?list=PLzdnOPI1iJNcXMS80NRhQd0NA3po1swJf"}
+    %img.dance_thumbnail-tile-img{src:"/images/creativity_playlist_thumbnail.png"}
+%h6.dance_thumbnail-tile-subtitle= I18n.t(:hoc2018_dance_view_creativity_video_playlist)
+
+.starwars-videos{style: "clear: both;"}
```
Comment (maddiedierker) [inline pegasus/sites.v3/code.org/public/dance.haml]: note: change from starwars to dance

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: @madelynkasula Finito!

File: pegasus/sites.v3/code.org/public/dance.haml (hunk 2 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: @madelynkasula Finito!

**Comment by nkiruka** [comment]
@madelynkasula All set, please review!

**Pull request merged**